### DISCUSSION TITLE
Export Mid+SF2 from Nintendo DS Music with...

### DISCUSSION THREAD

**Discussion by RetroGamesClub** [vgmtrans/vgmtrans#331] [Music formats]
I'll try again. Is it possible when exporting Nintendo DS Music, to add the loop point information inside the midi file to have the midi file loop correctly as almost all the nintendo DS music contain loop?
Thank you for your help.

**Comment by RetroGamesClub** [comment]
Any news? IN current release is it possible to see loop information inside Hex view, but generated midi files are without loop point. Please help.

**Reply by sykhro** [reply]
If there were news you'd see them in form of code being pushed. We are not very active at the moment as everyone is busy with other things. Community contributions are always welcome
